app/assets/javascripts/components/articles/PaginatedArticleControls.jsx

```diff
@@ -42,12 +42,17 @@ export const PaginatedArticleControls = ({ showMore, limitReached }) => {
       />) : null }
       {!limitReached
         && (
-          <button
-            style={{ width: 'max-content', height: 'max-content' }}
-            className="button ghost articles-see-more-btn" onClick={showMore}
-          >
-            {I18n.t('articles.see_more')}
-          </button>
+          <div className={'tooltip-trigger'}>
+            <button
+              style={{ width: 'max-content', height: 'max-content' }}
+              className="button ghost articles-see-more-btn" onClick={showMore}
+            >
+              {I18n.t('articles.see_more')}
+            </button>
+            <div className="tooltip tooltip-center tooltip-see-more dark large">
+              <p style={{ padding: '0px', textWrap: 'balance' }}>Click &apos;See more&apos; to load the next subset of 500 articles and continue the seamless navigation through the articles with the pagination on the left.</p>
```

Comment:
This should be implemented in the I18n system so that it can be translated.

Reply:
@PR4NJ41 you should add it in `en.yml` and ideally in `qqq.yml` as well. The latter is the documentation file, intended to describe the purpose and usage of the message, for translators, so they have enough context to translate it well. All the other locale files get handled via the volunteer translator community at translatewiki.net.